core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpSamDAO.scala

```diff
@@ -172,7 +172,7 @@ class HttpSamDAO(baseSamServiceURL: String, serviceAccountCreds: Credential, tim
         .resourceId(resourceId)
         .authDomain(authDomain.toList.asJava)
         .policies(policies.map { case (policyName, policy) =>
-          policyName.value -> new sam.model.AccessPolicyMembershipV2()
+          policyName.value -> new sam.model.AccessPolicyMembershipRequest()
```

Comment:
Updating this wasn't necessary, but I decided to go ahead and pull in the new Sam client because I have the context of what changed.